.github/workflows/package-workflow.yml

```diff
@@ -0,0 +1,53 @@
+name: E4E Pipeline
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      PROJECT_NAME: explorer-for-endevor
+
+    strategy:
+      # Continue to run tests on the other systems if one fails
+      fail-fast: false
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Audit
+        id: audit
+        continue-on-error: true
+        run: yarn audit
+
+      - name: Lint
+        run: yarn workspace explorer-for-endevor lint
+
+      - name: Run tests
+        uses: GabrielBB/xvfb-action@v1.0
+        with:
+          run: yarn workspace explorer-for-endevor test
+
+      - name: Build package
+        if: steps.audit.outputs.exit_code < 7
```

Comment:
tbh, I would specify the same `if` for all steps after audit, because this check is more for us to track, it should not block the pipeline, at least in my understanding

Reply:
What should not block the pipeline, the `if` statement? Or the audit?

The `if` statement/audit _will_ block the pipeline. The `if` statement will block every step on which it is applied, when the audit fails. So, I am confused.